### PR TITLE
[bin] adding check cli for mev

### PR DESCRIPTION
**Pull request by ochaloup**
I was adding a CLI command to check when the last MEV collection was run and work based on that.

This is connected to further `ops-infra` change of the k8s script running MEV collect and store.

### CODE REVIEW

File: check/src/validators_mev.rs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            let epoch_data = rpc_client.get_epoch_info()?;
+            let current_epoch = Decimal::from(epoch_data.epoch);
+            let current_slot = Decimal::from(epoch_data.absolute_slot);
+
+            info!(
+                "DB stores last MEV epoch: {sql_epoch}, slot: {sql_slot}, on-chain epoch: {current_epoch}, slot: {current_slot}"
+            );
+
+            // If the current epoch is bigger than the lastly stored MEV epoch, we need to proceed
+            if current_epoch > sql_epoch {
```
Comment (janlegner) [inline check/src/validators_mev.rs:35]: isn't this off by 1?

Comment (janlegner) [reply check/src/validators_mev.rs:35]: (also, linguistic nitpick: `greater`)

Comment (ochaloup) [reply check/src/validators_mev.rs]: Yes, you're right. Thanks. I haven't realized that while not having access to prod data. I updated the code.

**Comment by ochaloup** [comment]
I'm going to merge this as it does not impact any current processing of the delegation strategy until ops-infra is updated and docker built.
I would consider the ops-infra update to be kicked into live processing just at the start of next week (not now on Friday afternoon).

**Pull request merged**